frontend/src/metabase/querying/components/FilterPanel/utils.ts

```diff
@@ -11,6 +11,6 @@ export function getFilterItems(query: Lib.Query): FilterItem[] {
   });
 }
 
-export function dropStageIfEmpty(query: Lib.Query): Lib.Query {
-  return Lib.dropStageIfEmpty(query, -1);
+export function dropEmptyStages(query: Lib.Query): Lib.Query {
```

Comment:
We should inline this function as it no longer serves any purpose

Reply:
done